src/MetaMorpho.sol

```diff
@@ -472,11 +472,15 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
     }
 
     /// @inheritdoc IERC4626
+    /// @notice Returns the max amount of assets a user can withdraw without reverting.
```

Comment:
I think this one is redundant with `@inheritdoc IERC4626` which should already contains this description